src/background/redirects.spec.js

```diff
@@ -20,61 +20,85 @@ describe("Fandom redirect", () => {
-            expected: "https://www.google.com/search?q=site:poewiki.net+faster+attacks",
-        },
-        {
-            url: "https://www.google.com/search?q=poewiki+faster+attacks&rlz=1CDSA2EA_enUS653US116&oq=poe+test+wiki&aqs=chrome..6213i57j64.1j7&sourceid=chrome&ie=UTF-8",
-            expected: "https://www.google.com/search?q=site:poewiki.net+faster+attacks",
-        },
-        {
-            url: "https://google.com/search?q=poewiki+faster+attacks",
-            expected: "https://www.google.com/search?q=site:poewiki.net+faster+attacks",
-        },
-        // Wildcard *poe*wiki* cases
-        {
-            url: "https://www.google.com/search?q=poe+wiki+faster+attacks",
-            expected: "https://www.google.com/search?q=site:poewiki.net+faster+attacks",
-        },
-        {
-            url: "https://www.google.com/search?q=poe++wiki+faster+attacks",
-            expected: "https://www.google.com/search?q=site:poewiki.net+faster+attacks",
-        },
-        {
-            url: "https://www.google.com/search?q=poe+faster+attacks+wiki",
-            expected: "https://www.google.com/search?q=site:poewiki.net+faster+attacks",
-        },
-        {
-            url: "https://www.google.com/search?q=faster+poe+attacks+wiki",
-            expected: "https://www.google.com/search?q=site:poewiki.net+faster+attacks",
-        },
-        {
-            url: "https://www.google.com/search?q=faster+attacks+poe+wiki",
-            expected: "https://www.google.com/search?q=site:poewiki.net+faster+attacks",
-        },
-        {
-            url: "https://www.google.com/search?q=faster+attacks+poewiki",
-            expected: "https://www.google.com/search?q=site:poewiki.net+faster+attacks",
-        },
-        {
-            url: "https://www.google.com/search?q=faster+poewiki+attacks",
-            expected: "https://www.google.com/search?q=site:poewiki.net+faster+attacks",
-        },
-        // Making sure words that happen to contain "poe", "wiki" or "poewiki" do not get filtered
-        {
-            url: "https://www.google.com/search?q=poe+apoe+poea+apoea+awiki+wikia+awikia+wiki+apoewiki+poewikia+apoewikia+poeawiki",
-            expected: "https://www.google.com/search?q=site:poewiki.net+apoe+poea+apoea+awiki+wikia+awikia+apoewiki+poewikia+apoewikia+poeawiki",
-        },
-    ])("Given $url, redirect to $expected", ({ url, expected }) => {
-        const actual = redirectFromGoogle({ url })
-        expect(actual.redirectUrl).toBe(expected)
-    })
+describe("Test queries against all Search Engines redirect", () => {
+    //Setup different Search Engine urls to test against
+    const googleUrls = [
+        "https://www.google.com/search?",
+        "https://google.com/search?",
+    ]
+
+    const duckduckgoUrls = [
+        "https://duckduckgo.com/?",
+        "https://www.duckduckgo.com/?"
+    ]
+
+    //The base redirect result URL to expect from each search engine
+    const redirectHost = new Map()
```

Comment:
1. Why not just use a JS object? Map isn't necessary here as you're not dynamically changing these keys+values

2. nit - containers like these maps and objects should be plural (because they hold multiple elements)

3. nit - the keys of this container represent the hosts, but the *values* of this map are URLs that's more than just the host (not sure what I'd call them, perhaps "base"s?)

Here's how I'd approach this (you're welcome to disagree!):

```js
const redirectBaseUrls = {
  google: "https://www.google.com/search?q=site:poewiki.net+",
  duckduckgo: "https://www.duckduckgo.com/?q=site:poewiki.net+"
}
// .get becomes
redirectBaseUrls[searchEngine]
```